charts/osm/templates/osm-label.yml

```diff
@@ -137,4 +144,34 @@ spec:
                 else
                   echo "Failed to retrieve ${namespace} details"
                 fi
-              done 
+              done
+              echo "Retrieving namespace ${osmnamespace} details"
```

Comment:
Note: This job run can be denied by Azure Policy Add-on Gatekeeper if user assigns some of the policies that results this job in a violation. We can't do much in this case as we can't know all the policies that user installs.
As a safeguard, Azure Policy deployment should exclude arc-osm-system namespace in the charts. Can you please raise a PR in azure-policy-kubernetes repo adding arc-osm-system namespace in fullScanExclusionList and webhookExclusionList in https://github.com/Azure/azure-policy-kubernetes/blob/prod/charts/azure-policy-addon-arc-clusters/values.yaml ?